Agile_Essay.tex

```diff
@@ -0,0 +1,97 @@
+
+Subjects with introverted or neurotic personality types were noted as less likely to be inclined towards agile. Such findings are not unique to the Agile Development process, workplaces in general tend to favour those who are extroverted and open. As Susan Cain explains, \textit{`studies show that we rank fast and frequent talkers as more competent, likeable, and even smarter than slow ones' }\cite{Cain}. Such views are also seen in educational settings \textit{`research suggests that the majority of our teachers believe that the 'ideal student' is an extrovert.'} \cite{Cain}. This paper sets out to explore how introverted personality types can best adapt to Agile and how they can be aided in doing so.
+
+
+\section{Introversion, the Games Industry and Agile}
+
+It is often assumed that introverts are well suited to computing, \textit{`People in and out of the computing field often believe that a) technology is a field that is very attractive to introverts, and b) technology is a field that has traditionally been very amenable to introverts'} \cite{Kacz}, and certainly in early hacker culture, there is evidence to support such beliefs. Yet, in computing, especially within the games industry, teamwork has become increasingly important. Today, as opposed to the height of hacker culture, programming is conducted in teams, often large and evolving, and as such programming may not be as ideal to the introverted as it once was. 
+
+Indeed, introverted traits often contradict with the agile manifesto: they tend to enjoy working with few interruptions, prefer not to show, or discuss work before it is completed and feel drained after only small amounts of social activity\cite{Cain}.  These traits can make working within a team difficult, and within Agile, learning to manage such traits and preferences is essential.
+
+Whilst it could be argued that individuals who struggle with teamwork, are not suited to the game development industry, a multiplicity of personalities within a team has been found to be significantly beneficial,\textit{`The degree of homogeneity of personality of members of the groups used in this study was seen to have a direct bearing on the effectiveness of the groups in producing solutions to problems'}\cite{Hoffman}. A diverse range of personality types can enhance team performance and therefore, the whole team may benefit from supporting colleagues in adjusting to agile. Perhaps, it is important to remember that in adopting the Agile Manifesto, a team does not necessarily need to follow it as verbatim. Instead Agile can be adapted to teams by learning from both positive and negative experiences. 

```

Comment:
The sentence about `positive and negative experiences` seems to come out of nowhere and breaks the flow. Perhaps rephrase or integrate this differently into the paragraph to preserve the flow of the argument?